<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>565</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>565</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Baixa Aí do Youtube - darkmathew</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/youtube.png</normaloff>images/youtube.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Download">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(75, 75, 75);
border-style: outset;
border-width: 1px;
border-radius: 4px;
border-color: black;
padding: 2px;
}

QPushButton:hover:!pressed
{
  border-color: rgb(23, 205, 255);
  padding: 2px;
  border-radius: 4px;
}
</string>
    </property>
    <property name="text">
     <string>INICIAR DOWNLOAD</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/download.png</normaloff>images/download.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_youtubeLink">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
	color: rgb(255, 255,255);
}
QLineEdit:hover {
	border-color: rgb(23, 205, 255);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Insira aqui o link do Youtube</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_activities">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
background-color: rgb(35, 35, 35);
border-style: outset;
border-width: 1px;
border-radius: 4px;
padding: 5px;
}

QTextEdit:hover:!pressed
{
  border-color: rgb(23, 205, 255);
  padding: 5px;
  border-width: 2px;
  border-radius: 5px;
}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>393</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-radius: 6px;
	background-color: rgb(38, 38, 38);
	border: 1px solid black;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	color: rgb(177, 177, 177);
	border: none;
	background-color: rgb(102, 102, 102);
	border-radius: 6px;
}

QProgressBar::chunk {
background-color:#00f2ff	;
border-radius : 6px;
}     </string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
	color: rgb(255, 255,255);
}
QLineEdit:hover {
	border-color: rgb(23, 205, 255);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Local para salvar os arquivos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openFileDialog">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color:  rgb(27, 29, 35);
border-style: outset;
border-width: 1px;
border-radius: 4px;
border-color: black;
padding: 2px;
}

QPushButton:hover:!pressed
{
  border-color: rgb(23, 205, 255);
  padding: 2px;
  border-radius: 4px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/folder.png</normaloff>images/folder.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
color: rgb(255, 255, 255);
background-color: rgb(27, 29, 35);
border-radius: 5px;
border: 2px solid rgb(33, 37, 43);
padding: 5px;
padding-left: 10px;
}

QComboBox::drop-down {

}

QComboBOx:QScrollBar:vertical {
width: 3px;
}

QComboBox:hover{
border: 2px solid rgb(23, 205, 255);
}

QComboBox:focus {
border: 2px solid rgb(91, 101, 124);
}


QComboBox QAbstractItemView {
color: rgb(255, 255, 255);
background-color: rgb(33, 37, 43);
padding: 10px;
selection-background-color: rgb(39, 44, 54);
}</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Baixar apenas em formato mp3</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/notas-musicais.png</normaloff>images/notas-musicais.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baixar vídeo [Qualidade Alta] (arquivo maior)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/video.png</normaloff>images/video.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baixar vídeo [Qualidade Baixa] (arquivo menor)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/video.png</normaloff>images/video.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baixar vídeo [Qualidade Alta] + mp3</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/botao-play.png</normaloff>images/botao-play.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baixar vídeo [Qualidade Baixa] + mp3</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/botao-play.png</normaloff>images/botao-play.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{ 
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>Desenvolvido por: https://github.com/darkmathew</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
